### DISCUSSION TITLE
Encountered errors while installing CS tools

### DISCUSSION THREAD

**Discussion by Johnas-Castro** [thoughtspot/cs_tools#140] [Q&A]
PLs help.
Encountered errors while installing CS tools
![image](https://github.com/thoughtspot/cs_tools/assets/170397007/273eae2a-1f32-4396-b0f5-637e0c8ff4dd)

**Comment by boonhapus** [comment]
@Johnas-Castro can you re-run the command with `--verbose` and see if it produces any better information?

`powershell -c "IRM https://raw.githubusercontent.com/thoughtspot/cs_tools/master/cs_tools/updater/_bootstrapper.py | python - --install --verbose"`
